src/Documentation.md

```diff
@@ -70,6 +70,13 @@ import DataExplorer from "@nteract/data-explorer";
   initialView="summary"
   metadata={{
     dx: {
+      overrideSettings: {
+        backgroundGraphics: <text
+          fontSize="24px"
+          fontWeight={900}
+          fill="#DDD"
+          y={20}>DX™</text>
+      },
```

Comment:
One change this would make is that `metadata` would no longer be json serializable if it can take arbitrary jsx. This would make it harder to store in a notebook file or `localStorage` etc if users wanted to keep states across page loads.  

What do you think about moving  `backgroundGraphics` to its own prop outside of metadata?

Reply:
Good point, I'll move it all the way to the top, that's how I intended it to work (someone could extend DX in a different environment with whatever additional properties) and not that someone could save these settings on a per-notebook basis.